lib/index.js

```diff
@@ -1,53 +1,65 @@
 let path = require("path");
-let { readFile } = require("./promisified-fs");
+let { readFile, stat } = require("./promisified-fs");
 let tree = require("./tree");
 
 module.exports = (pluginConfig, assetManager, { watcher }) => {
-	let copyAll = buildCopyAll(pluginConfig, assetManager);
+	buildCopyAll(pluginConfig, assetManager).then(copyAll => {
+		// Run once for all files
+		copyAll();
 
-	// Run once for all files
-	copyAll();
-
-	if(watcher) {
-		watcher.on("edit", copyAll);
-	}
+		if(watcher) {
+			watcher.on("edit", copyAll);
+		}
+	});
 };
 
 function buildCopyAll(copyConfigs, assetManager) {
-	let copiers = copyConfigs.map(copyConfig =>
+	let futureCopiers = copyConfigs.map(copyConfig =>
 		buildCopier(copyConfig, assetManager));
 
-	return files => copiers.forEach(copier => copier(files));
+	return Promise.all(futureCopiers).then(copiers => {
+		return files => copiers.forEach(copier => copier(files));
+	});
 }
 
 function buildCopier(copyConfig, assetManager) {
 	let source = assetManager.resolvePath(copyConfig.source, {
-		enforceRelative: true
+		enforceRelative: false
```

Comment:
FWIW, I don't like 015d913f27d9e577fad6a29dcf5b3185ee8aff43's commit message:

>  Fix #1
>  Allow people to copy files from their packages with absolute paths

I'd swap both lines because the first one is almost meaningless.